client/engine/chainservice/eth_chainservice.go

```diff
@@ -173,10 +177,10 @@ func (ecs *EthChainService) fatalF(format string, v ...any) {
 
 	// Print to STDOUT in case we're using a noop logger
 	fmt.Println(fmt.Errorf(format, v...))
-	// FatalF prints to the logger then calls exit(1)
-	ecs.logger.Fatalf(format, v...)
 
-	// Manually panic in case we're using a logger that doesn't call exit(1)
+	ecs.logger.Printf(format, v...)
```

Comment:
Should we use panic log level here?

Reply:
[2a6f452](https://github.com/statechannels/go-nitro/pull/1070/commits/2a6f45233972f53e0cac1d3b41dcc9936d55b96f)